pkg/patches/patch.go

```diff
@@ -46,7 +46,7 @@ func ApplyPatches(obj1, obj2 client.Object, patchesConf []*config.Patch, reverse
 
 	// remove ignore paths from patched object
 	for _, p := range reversePatchesConf {
-		if p.Path == "" || (p.Ignore != nil && !*p.Ignore) {
```

Comment:
Not sure if that is correct, the logic currently is that its true by default, this would make it false by default, so only in the case of `ignore: false` we want to skip applying the delete patch